### ISSUE TITLE
[Improvement] Generate localized BAT file for improved consistency.

### ISSUE THREAD

**Issue by ParasiteDelta** [derrod/rktlnch#3]
As per my last Issue, my game would instantly crash with a "Your graphics card or driver does not support DirectX 11" (see closed Issue #2), and after a lot of scrapping and tinkering with Python (since I don't know Python), I crafted a solution. Admittedly, it's rather janky, but it should work 100% of the time. Rather than directly executing the game EXE with the arguments, I changed it to parse, modify, and push all that it needs into a BAT file which is then placed directly into the game folder containing the EXE. Every single time that I would run the BAT file outside of the Python program, it would work. Every time that I tried running the BAT file through a subprocess, it would fail. For some reason, saving the data into a BAT located in the same folder as the game EXE and then running it via subprocess.Popen works flawlessly.

I can upload my changes somewhere if the OA wants them.

**Comment by derrod** [comment]
This project is not really maintained since it was more of a PoC before I started working on [legendary](https://github.com/derrod/legendary). And that "solution" (I barely dare call it that) seems extremely janky and unnecessary and no patches doing anything like that will be accepted. However if you can figure out what the *actual* issue is then please go ahead.

**Comment by ParasiteDelta** [comment]
While I normally wouldn't bother responding to such an egotistical reply, I found the solution minutes after reviewing said reply. It seems that certain games such as World War Z may have some form of permissions error for execution, where it just won't run correctly. How I solved this was by piecing together the path containing the main game EXE and running the command from that folder with no external files, which made it work 100% of the time. I enjoyed using it for about a fortnight before it suddenly quit working with a CSE-400 Bad Request. No idea why, but Legendary does indeed work.

Overall, hope you piece it together.
Signed, A Tinkerer.
